### ISSUE TITLE
Feature Request: Add additional package manager support for AWS CodeArtifact

### ISSUE THREAD

**Issue by charles545587** [projen/projen#1760]
[AWS Code Artifact currently supports npm, PyPI, Maven and NuGet package formats](AWS CodeArtifact supports [npm](https://docs.aws.amazon.com/codeartifact/latest/ug/using-npm.html), [PyPI](https://docs.aws.amazon.com/codeartifact/latest/ug/using-python.html), [Maven](https://docs.aws.amazon.com/codeartifact/latest/ug/using-maven.html), and [NuGet](https://docs.aws.amazon.com/codeartifact/latest/ug/using-nuget.html) package formats.). Projen [publisher](https://projen.io/publisher.html) currently only supports publishing to AWS CodeArtifact using npm.  

This feature request is for Projen to add publishing support for PyPI, Maven and NuGet package formations using AWS CodeArtifact.

**Comment by mraza007** [comment]
👍

**Comment by Chriscbr** [comment]
Yes this would be a welcome feature. Pull requests are welcome. 👍 

Projen currently uses [publib](https://github.com/cdklabs/publib) for publishing generated packages to different package managers, so support for CodeArtifact may need to be added there first.
